recipes/examples/movies_trending.yml

```diff
@@ -14,12 +14,28 @@ source_list_urls:
 # Source library details
 source_libraries:
   - name: 'Movies'
-  - name: 'Movies (4K)'
+    folders:
+      - '/path/to/Movies'
+      - '/path/to/More Movies'
+  - name: 'Different Movies'
+    folders:
+      - '/path/to/Different Movies'
 
 # New playlist details
 new_playlist:
```

Comment:
Could we unify `new_playlist` and `new_library` somewhat? I realize that might require some migration of existing recipes (but we could be backwards compatible). Right now there's lots of duplication and getting a bit confusing.

Reply:
What would you suggest? Place the settings of both ``new_playlist`` and ``new_library`` under one category? Seems ``name``, ``max_age``, ``max_count`` and ``remove_from`` are the duplicated settings.

Reply:
Something like that, or perhaps just the duplicated settings. But honestly it can wait.